### PR TITLE
task(FDS-524): sets selection context via selections prop

### PR DESCRIPTION
**Pull request by ricardogonz**
Co-authored-by: Manu Ramirez <manu.ramirez@espressive.com>## PR Author Checklist

- [x] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
- [x] I have finished reviewing the contents of this PR for accuracy
- [x] If needed, I have included a [Changeset](https://github.com/changesets/changesets) and it follows [Semantic Versioning principles](https://semver.org/)
- [x] I have completed all of the above and have enabled `auto-merge` for this PR

### CODE REVIEW

**Comment by mementablog** [comment]
LGTM

**Comment by EspressiveGithub** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  **packages/cascara/src/components/Table/fixtures/Selection.fixture.js**: This fixture has been changed.
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0bca7b9fc4a98400e001456037b8dfcd73469130
</p>

File: packages/cascara/src/components/Table/fixtures/Selection.fixture.js [hunk 1]
```diff
@@ -44,4 +44,22 @@ export default {
...
   ),
+  withSelectedRows: (
+    <DataWithDisplay
+      actions={ACTIONS}
+      data={results}
+      dataDisplay={COLUMNS}
+      description='Selection can be limited to a given number of records'
+      onAction={console.log}
+      selections={{
+        selectedRows: [
```
Comment (mementablog) [inline packages/cascara/src/components/Table/fixtures/Selection.fixture.js:55]: 👍

**Review by mementablog — APPROVED** [review]

File: packages/cascara/src/components/Table/fixtures/Selection.fixture.js (hunk 1 above)
Comment (ricardogonz) [reply packages/cascara/src/components/Table/fixtures/Selection.fixture.js:55]: :)

**Review by odiseo42 — APPROVED** [review]
LGTM

**Pull request merged**